pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+
+  lib,
+
+  buildGoModule,
+
+  fetchFromGitHub,
+
+}:
+
+buildGoModule rec {
+  pname = "gf";
+  version = "2.9.0";
+
+  src = fetchFromGitHub {
+    owner = "cococolanosugar";
+    repo = "gf";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```

There is a chance that upstream will name a tag with the same name as a `git` HEAD, which can cause issues with this fetcher's fixed output derivation if upstream pushes to that branch. Since we want the tags, which are usually less prone to change, we can use the `tag` attribute that automatically prefixes the value with `refs/tags`, preventing the fetcher from fetching anything that's not a tag.